### PR TITLE
Add frogs command

### PR DESCRIPTION
**Pull request by fbalarini**
I added a command with useful information about Frogs. There is also a link to a form to give feedback about the new website.

### CODE REVIEW

File: src/com/xmartlabs/slackbot/CommandManager.kt [hunk 1]
```diff
@@ -153,8 +153,23 @@ object CommandManager {
             """
                Hey, thanks for sharing your feedback! :muscle:
                If you want to propose a new feature you can open a <https://github.com/xmartlabs/slackbot/discussions | GitHub Discussion> or an <https://github.com/xmartlabs/slackbot/issues | issue>.
-               
-               All contributions are welcomed! :github: 
+
+               All contributions are welcomed! :github:
+            """.trimIndent()
+        },
+        Command(
```
Comment (mirland) [inline src/com/xmartlabs/slackbot/CommandManager.kt:160]: I would locate it above the slack command, so the slack and feedback commands are the last commands. What do you think?

File: src/com/xmartlabs/slackbot/CommandManager.kt [hunk 2]
```diff
@@ -153,8 +153,23 @@ object CommandManager {
...
+            "frogs",
+            title = "Frogs information :frog:",
+            description = "Give me more information about Frogs :pray:"
+        ) { _, _ ->
+            """
+               Hey, here there is useful information about Frogs:
+               * If you have no idea what is Frogs and how it works, check this <https://www.notion.so/xmartlabs/Frogs-2668aa7ab5d64f0d99f16d4233e6d5fa | link>.
+               * We have new website! Check it out at https://frogs.xmartlabs.com/
```
Comment (mirland) [inline src/com/xmartlabs/slackbot/CommandManager.kt:168]: Maybe in a link format? What do you think?
```suggestion
               * We have new website! Check it out at <https://frogs.xmartlabs.com/ | frogs.xmartlabs.com>
```

File: src/com/xmartlabs/slackbot/CommandManager.kt [hunk 3]
```diff
@@ -153,8 +153,23 @@ object CommandManager {
...
+            """.trimIndent()
+        },
+        Command(
+            "frogs",
+            title = "Frogs information :frog:",
+            description = "Give me more information about Frogs :pray:"
+        ) { _, _ ->
+            """
+               Hey, here there is useful information about Frogs:
+               * If you have no idea what is Frogs and how it works, check this <https://www.notion.so/xmartlabs/Frogs-2668aa7ab5d64f0d99f16d4233e6d5fa | link>.
```
Comment (mirland) [inline src/com/xmartlabs/slackbot/CommandManager.kt:167]: Use • to be consistent with the previous comment

**Review by mirland — APPROVED** [review]
Cool

File: src/com/xmartlabs/slackbot/CommandManager.kt (hunk 1 above)
Comment (fbalarini) [reply src/com/xmartlabs/slackbot/CommandManager.kt:160]: sounds good

File: src/com/xmartlabs/slackbot/CommandManager.kt (hunk 2 above)
Comment (fbalarini) [reply src/com/xmartlabs/slackbot/CommandManager.kt:168]: sure!

File: src/com/xmartlabs/slackbot/CommandManager.kt (hunk 3 above)
Comment (fbalarini) [reply src/com/xmartlabs/slackbot/CommandManager.kt:167]: instead of *?

**Pull request merged**